### ISSUE TITLE
Details about pretraining on CT-3M

### ISSUE THREAD

**Issue by kwinderic** [ricklisz/MedDINOv3#7]
Hi, did you use labels together with images or images only when pretraining your model?

Thanks.

**Comment by ricklisz** [comment]
For pretraining we only use unlabeled CT volumes which are sliced into 2D images.

**Comment by sayedmohamedscu** [comment]
Could you please share the config / yaml file or the way you followed to finetune the dinov3?

**Comment by ricklisz** [comment]
Sure. I used the yaml file for vitl_im1k_lin834.yaml  and the only change I made is to switch vit_large to vit_base.
